### ISSUE TITLE
Would love to use this plugin but having a bit of trouble

### ISSUE THREAD

**Issue by planetClaire** [wpmotto/wp-blazing-css#4]
This plugin looks great, thank you.

Trying to run locally using VVV on Windows 10

Firstly, I had to run `composer install` in the plugin folder, would this need to be done on the server? (not a PHP buff)

Secondly - need to load pages while not logged in or nothing happens - this took me a while to figure out, you might want to document it...

Finally, I'm trying to debug this without much luck but getting exceptions when loading my home page:

```
[10-Aug-2021 01:45:29 UTC] PHP Notice:  Undefined property: stdClass::$query in /srv/www/nrpg/public_html/wp-content/plugins/wp-blaze-css/lib/Common/Logger.php on line 102
[10-Aug-2021 01:45:29 UTC] PHP Notice:  Trying to get property 'id' of non-object in /srv/www/nrpg/public_html/wp-content/plugins/wp-blaze-css/lib/Common/Model.php on line 102
[10-Aug-2021 01:45:29 UTC] PHP Notice:  wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /srv/www/nrpg/public_html/wp-includes/functions.php on line 5535
[10-Aug-2021 01:45:29 UTC] PHP Notice:  Undefined offset: 0 in /srv/www/nrpg/public_html/wp-includes/wp-db.php on line 1323
[10-Aug-2021 01:45:29 UTC] PHP Notice:  Undefined property: stdClass::$query in /srv/www/nrpg/public_html/wp-content/plugins/wp-blaze-css/lib/Common/Logger.php on line 102
[10-Aug-2021 01:45:30 UTC] PHP Notice:  wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /srv/www/nrpg/public_html/wp-includes/functions.php on line 5535
[10-Aug-2021 01:45:30 UTC] PHP Notice:  Undefined offset: 0 in /srv/www/nrpg/public_html/wp-includes/wp-db.php on line 1323
[10-Aug-2021 01:45:30 UTC] PHP Notice:  Undefined property: stdClass::$query in /srv/www/nrpg/public_html/wp-content/plugins/wp-blaze-css/lib/Common/Logger.php on line 102

```

**Comment by greghunt** [comment]
Hi @planetClaire, the installable plugin is available directly in from the [WordPress repo](https://wordpress.org/plugins/blaze-css/). This will include the composer packages and not require any `composer install`. I suggest installing from there and not here, since the master branch on this repo is currently just for development and not as stable. I also make mention of visiting pages there.

Please try the installation from the WP repo and let me know if you still have PHP notices. I've released some fixes to that end.

**Comment by planetClaire** [comment]
Hi @greghunt I did install this from WP repo actually.  I just tried again & on activating got 

`Fatal error: Uncaught Error: Class 'Motto\BlazeCss\Plugin' not found in /srv/www/wordpress-one/public_html/wp-content/plugins/blaze-css/blaze-css.php:55 Stack trace: #0 /srv/www/wordpress-one/public_html/wp-includes/class-wp-hook.php(303): {closure}() #1 /srv/www/wordpress-one/public_html/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters() #2 /srv/www/wordpress-one/public_html/wp-includes/plugin.php(470): WP_Hook->do_action() #3 /srv/www/wordpress-one/public_html/wp-settings.php(441): do_action() #4 /srv/www/wordpress-one/public_html/wp-config.php(82): require_once('/srv/www/wordpr...') #5 /srv/www/wordpress-one/public_html/wp-load.php(50): require_once('/srv/www/wordpr...') #6 /srv/www/wordpress-one/public_html/wp-admin/admin.php(34): require_once('/srv/www/wordpr...') #7 /srv/www/wordpress-one/public_html/wp-admin/plugins.php(10): require_once('/srv/www/wordpr...') #8 {main} thrown in /srv/www/wordpress-one/public_html/wp-content/plugins/blaze-css/blaze-css.php on line 55`

After running composer install the plugin activates & restores the website, but now there are logs like the following after visiting public pages
```

[20-Aug-2021 05:29:33 UTC] PHP Notice:  Trying to get property 'id' of non-object in /srv/www/wordpress-one/public_html/wp-content/plugins/blaze-css/lib/Common/Model.php on line 102
[20-Aug-2021 05:29:33 UTC] PHP Notice:  wpdb::prepare was called <strong>incorrectly</strong>. The query argument of wpdb::prepare() must have a placeholder. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 3.9.0.) in /srv/www/wordpress-one/public_html/wp-includes/functions.php on line 5535
[20-Aug-2021 05:29:33 UTC] PHP Notice:  Undefined offset: 0 in /srv/www/wordpress-one/public_html/wp-includes/wp-db.php on line 1323
[20-Aug-2021 05:29:33 UTC] WordPress database error Table 'wordpress-one.wp_blaze_elements' doesn't exist for query SELECT DISTINCT el_class 
FROM wp_blaze_elements            
WHERE el_class IS NOT NULL  made by do_action('wp_ajax_nopriv_blaze_ajax'), WP_Hook->do_action, WP_Hook->apply_filters, Motto\BlazeCss\Frontend->save_page_elements, Motto\BlazeCss\Common\Logger->save, Motto\BlazeCss\Common\File::write, Motto\BlazeCss\Common\Model->get
[20-Aug-2021 05:29:33 UTC] PHP Notice:  Undefined property: stdClass::$query in /srv/www/wordpress-one/public_html/wp-content/plugins/blaze-css/lib/Common/Logger.php on line 102
```

I haven't tried activating the plugin on a live website.

**Comment by greghunt** [comment]
Fixed in `1.1.3`
